runtime/tasks.py

```diff
@@ -0,0 +1,56 @@
+from django_rq.decorators import job
+
+from predModels.models import PredModels
+from logs.models import Log, Split
+from logs.file_service import get_logs
+from core.classification import classifier_single_log
+from core.next_activity import next_activity_single_log
+from core.regression import regression_single_log
+from encoders.common import encode_log
+from jobs.models import Job, CREATED, RUNNING, COMPLETED, ERROR
+from core.constants import CLASSIFICATION, NEXT_ACTIVITY, REGRESSION
+from core.core import get_run
+
+@job("default", timeout='1h')
+def runtime_task(job, model):
+    print("Start runtime task ID {}".format(job.pk))
+    try:
+        if job.status == CREATED:
+            job.status = RUNNING
+            job.save()
+            log = Log.objects.get(pk=job['log_id'])
+            run_log = get_logs(log.path)[0]
+            result = calculate(run_log,model.to_dict())
+            job.result = result
+            job.status = COMPLETED
+    except Exception as e:
+        print("error " + str(e.__repr__()))
+        job.status = ERROR
+        job.error = str(e.__repr__())
+        raise e
+    finally:
+        job.save()
+        
+def calculate(run_log,model):
```

Comment:
This method should be in core/core.py

Reply:
It's good to have all the calculating logic in one package.

Reply:
I thought we wanted to fully separate our code, but if you say so I'll put them in the core folder

Reply:
This method at least as there's nothing here that's specifically for runtime package.